src/index.ts

```diff
@@ -36,14 +37,34 @@ import { build as buildNexusPrismaTypes, Options } from './builder'
  * class support for typegen.
  */
 export function nexusPrismaPlugin(options?: Options) {
+  let wasCrudUsedButDisabled: null | (() => boolean) = null
+
   return plugin({
     name: 'nexus-prisma',
     onInstall: nexusBuilder => {
+      const {
+        types,
+        wasCrudUsedButDisabled: wasCrudUsed,
+      } = buildNexusPrismaTypes({
+        ...options,
+        nexusBuilder,
+      })
+
+      wasCrudUsedButDisabled = wasCrudUsed
+
       return {
-        types: buildNexusPrismaTypes({
-          ...options,
-          nexusBuilder,
-        }),
+        types,
+      }
+    },
+    onBeforeBuild() {
+      if (wasCrudUsedButDisabled?.() === true) {
+        console.log(`\
+${colors.yellow('WARNING')}: ${colors.green(
+          't.crud',
+        )} ${colors.yellow('is an experimental feature that may evolve and break in the future. It must be explicitely enabled to be used.')}
+Please add ${colors.green(`experimentalCRUD: true`)} in the ${colors.green(
+          'nexusPluginPrisma()',
+        )} if you still wish to enable it.`)
       }
```

Comment:
Core addition of this PR. If you have suggestions to improve the message, I'll happily change it